package.json

```
@@ -28,11 +28,10 @@
     "licensee"
   ],
   "devDependencies": {
-    "aud": "^2.0.4",
-    "rimraf": "^3.0.2",
+    "rimraf": "^4.3.1",
     "run-parallel": "^1.2.0",
     "spawn-sync": "^2.0.0",
-    "standard": "^14.3.4",
+    "standard": "^17.1.0",
```

```suggestion
    "standard": "^14.3.4",
```

Leave Standard be. It'll just complain about a bunch of old syntax.

---

It doesnt complain when I run it locally.

---

```shellsession
$ npx standard@17.1.0 
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Some warnings are present which will be errors in the next version (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /home/kyle/licensee.js/index.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:4:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:5:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:6:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:7:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:8:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:9:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:10:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:11:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:21:9: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:36:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:39:9: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:66:9: Expected newline between consequent and alternate of ternary expression. (multiline-ternary)
  /home/kyle/licensee.js/index.js:68:1: Expected indentation of 10 spaces but found 8. (indent)
  /home/kyle/licensee.js/index.js:73:1: Expected indentation of 8 spaces but found 6. (indent)
  /home/kyle/licensee.js/index.js:86:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:92:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:94:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:114:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:115:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:128:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:143:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:153:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:155:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:183:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:200:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:201:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:208:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:210:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:252:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:253:8: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:254:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:269:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:270:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:271:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/index.js:273:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/allowed/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/allowed/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/apache-2.0-mit-allowed/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/apache-2.0-mit-allowed/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-fail/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-fail/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-flag/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-flag/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-gold-mit/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-gold-mit/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-misspelled/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-misspelled/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-pass/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/blue-oak-pass/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-author/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-author/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-prefix/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-prefix/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-scope/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/ignored-scope/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/licenses-array-with-corrections/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/licenses-array-with-corrections/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/mit-not-allowed/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/mit-not-allowed/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/no-allowlist/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/no-allowlist/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/no-dependencies/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-with-corrections/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-with-corrections/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-with-corrections-in-dotfile/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-with-corrections-in-dotfile/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-without-corrections/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/optimist-without-corrections/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-fail/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-fail/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-flag-fail/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-flag-fail/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-flag-pass/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-flag-pass/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-pass/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/osi-pass/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/out-of-allowed-range/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/out-of-allowed-range/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/production-only/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/production-only/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/run.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/run.js:6:30: Expected property shorthand. (object-shorthand) (warning)
  /home/kyle/licensee.js/tests/run.js:7:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/run.js:7:52: Expected property shorthand. (object-shorthand) (warning)
  /home/kyle/licensee.js/tests/symlinked-node-modules/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/symlinked-node-modules/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:2:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:4:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:5:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:6:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:7:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:12:5: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:36:9: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unit.test.js:64:3: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unlicensed-subdependency/test.js:1:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unlicensed-subdependency/test.js:3:1: Unexpected var, use let or const instead. (no-var) (warning)
  /home/kyle/licensee.js/tests/unlicensed-subdependency/test.js:7:1: Unexpected var, use let or const instead. (no-var) (warning)
```